tutorials/navigation/navigation_setup_2d_scene.rst

```diff
@@ -0,0 +1,85 @@
+.. _doc_navigation_setup_2d_scene:
+
+Setup for 2D scene
+==========================================
```

Comment:
Number of `=` should match the title (this is a reStructuredText requirement):

```suggestion
Setup for 2D scene
==================
```

Same for `-` and `^` headings.